### DISCUSSION TITLE
What's the best way to synchronise settings?

### DISCUSSION THREAD

**Discussion by kud** [rxhanson/Rectangle#942] [Q&A]
Hey,

I've read the documentation but it seems not so clear to me.

There's `~/Library/Preferences/com.knollsoft.Rectangle.plist` but also `~/Library/Application Support/Rectangle/RectangleConfig.json`.

What should I do to synchronise my settings on multiple laptops?

Thank you,

**Comment by rxhanson** [comment]
The reason to use the json location is so that it could be symlinked and just the json for Rectangle can be automatically added there (better than tinkering with a location where other applications are affected). If you don't care about that, and/or don't care about json, then it's fine to just do the plist.
